h5c/vic/src/vic-webapp/src/app/interfaces/vch.ts

```diff
@@ -237,11 +251,12 @@ export interface VchApiContainer {
   name_convention?: string
 }
 
-
+// VCH Generals
 export interface VchUi {
   general?: VchUiGeneral;
   computeCapacity?: VchUiCompute;
+  storageCapacity?: VchUiStorage;
```

Comment:
yes, that is because now we could have a payload with only one property in the reconfigure flow (for instance if you are configuring the general page, or the computeCapacity page) or with all the properties on the creation flow. The new CLI component will receive this payload with all the properties or with a single one depending on if you are creating or configuring a vch.